config/locales/diaspora/en.yml

```diff
@@ -1106,6 +1106,8 @@ en:
   shared:
     aspect_dropdown:
       add_to_aspect: "Add contact"
+      mobile_row_checked: "✓ %{name} (remove)"
+      mobile_row_unchecked: "– %{name} (add)"
```

Comment:
will do. My thought was just "if we do string concatenation, do it all at once".